### PR TITLE
Release v1.6.3

### PR DESCRIPTION
**Pull request by asmacdo**
Signed-off-by: austin <austin@redhat.com>

**Description of the change:**

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: website/content/en/docs/installation/_index.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ export OS=$(uname | awk '{print tolower($0)}')
 Download the binary for your platform:
 
 ```sh
-export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.6.2
+export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/v1.6.3
```
Comment (estroz) [inline website/content/en/docs/installation/_index.md:39]: Release script needs to be fixed
```suggestion
export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.6.3
```

File: website/content/en/docs/upgrading-sdk-version/v1.6.3.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: v1.6.3
+weight: 998993997
+---
+
+## Ensure that existing dependent resources have owner annotations
+
+Ansible-based operators use owner references (for resources in the same namespace) and owner annotations(for cluster scoped resources and resources in other namespaces). If dependent resources were created without the references/annotations, they cannot be cleaned up by the operator, and update/deletes will not trigger a reconcile. Follow these instructions to add owner references/annotations to existing objects. https://sdk.operatorframework.io/docs/building-operators/ansible/reference/retroactively-owned-resources/
```
Comment (estroz) [inline website/content/en/docs/upgrading-sdk-version/v1.6.3.md:8]: Taken from master
```suggestion
For Ansible-based operators, [owner reference annotations](https://sdk.operatorframework.io/docs/building-operators/ansible/reference/retroactively-owned-resources/)
on cluster-scoped dependent resources and dependent resources in other namespaces were not applied correctly.
A workaround was to add these annotations manually, which is no longer required as this bug has been fixed.
```

**Review by estroz — APPROVED** [review]
/lgtm

**Review by jmrodri — APPROVED** [review]
/lgtm

**Pull request merged**